src/Console/PrintSchemaCommand.php

```diff
@@ -39,7 +40,7 @@ public function handle(Repository $cache, Filesystem $storage): void
         $cache->forget(config('lighthouse.cache.key'));
 
         $schema = SchemaPrinter::doPrint(
-            graphql()->prepSchema()
+            Helper::graphql()->prepSchema()
```

Comment:
Let's use proper DI instead of this one, really don't like the way it masks how it actually get's a singleton class under the hood.

Reply:
Just take `GraphQL` as a parameter in the args